src/app/components/Footer/index.tsx

```diff
@@ -84,24 +122,15 @@ export const Footer = memo(() => {
               ) : (
                 <NoReleaseLink />
               ),
-              CommitLink: (
-                <Anchor
-                  href={`${githubLink}commit/${process.env.REACT_APP_BUILD_SHA}`}
-                  label={process.env.REACT_APP_BUILD_SHA.substring(0, 7)}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                />
-              ),
-            }}
-            defaults="Version: <ReleaseLink/> (commit: <CommitLink/>) built at {{buildTime}}"
-            values={{
-              buildTime: intlDateTimeFormat(Number(process.env.REACT_APP_BUILD_DATETIME)),
+              CommitLink: <CommitLink />,
             }}
+            defaults="Version: <ReleaseLink/> <CommitLink/>"
           />
-          {poweredByLabel && <Box align="center">{poweredByLabel}</Box>}
```

Comment:
Linters should throw an error because `poweredByLabel` is defined, but not it is not used.